akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/ActorContext.scala

```diff
@@ -167,7 +167,7 @@ trait ActorContext[T] extends akka.actor.typed.ActorContext[T] { this: akka.acto
    * *Warning*: This method is not thread-safe and must not be accessed from threads other
    * than the ordinary actor message processing thread, such as [[scala.concurrent.Future]] callbacks.
    */
-  def setReceiveTimeout(d: FiniteDuration, msg: T): Unit
+  def setReceiveTimeout(receiveTimeout: FiniteDuration, msg: T): Unit
```

Comment:
perhaps `timeout: ` to not repeat too much from the method name

Reply:
I aliased it with the latter `responseTimeout` in the `ask` method.

Reply:
@patriknw Done.